File: swap/swap.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b, c;
    cout << "Enter three variables: ";
    cin >> a >> b >> c;
    cout << "Before Swapping:\na = " << a << "\nb = " << b << "\nc = " << c;
    a = a ^ b;
    b = a ^ b;
    a = a ^ b;
    b = b ^ c;
    c = b ^ c;
    b = b ^ c;
    cout << "After Swapping:\na = " << a << "\nb = " << b << "\nc = " << c;

    return 0;
}

#include <iostream>
using namespace std;
int main()
{
    int a, b, c;
    cout << "Enter three numbers: ";
    cin >> a >> b >> c;
    cout << "Before Swapping:\na = " << a << "\nb = " << b << "\nc = " << c;
    a = a + b + c;
    b = a - b - c;
    c = a - b - c;
    a = a - b - c;
    cout << "After Swapping:\na = " << a << "\nb = " << b << "\nc = " << c;
    return 0;
}

File: swap/swap.cs
// using System;

// namespace Swap
// {
// class Program
// {
// 	static void Main(string[] args)
// 	{
// 		int a, b, c;
// 		Console.Write("Enter 1st number: ");
// 		a = int.Parse(Console.ReadLine());
// 		Console.Write("Enter 1st number: ");
// 		b = int.Parse(Console.ReadLine());
// 		Console.Write("Enter 1st number: ");
// 		c = int.Parse(Console.ReadLine());

// 		Console.WriteLine("Before Swapping:");
// 		Console.WriteLine("a = {0}, b = {1}, c = {2}", a, b, c);
// 		a = a ^ b;
// 		b = a ^ b;
// 		a = a ^ b;
// 		b = b ^ c;
// 		c = b ^ c;
// 		b = b ^ c;
// 		Console.WriteLine("After Swapping:");
// 		Console.WriteLine("a = {0}, b = {1}, c = {2}", a, b, c);
// 	}
// }
// }



using System;

namespace Swap
{
    class Program
    {
        static void Main(string[] args)
        {
            int a,
                b,
                c;
            Console.Write("Enter 1st number:");
            a = int.Parse(Console.ReadLine());
            Console.Write("Enter 2nd number: ");
            b = int.Parse(Console.ReadLine());
            Console.Write("Enter 3rd number: ");
            c = int.Parse(Console.ReadLine());
            Console.WriteLine("Before Swapping: ");
            Console.WriteLine("a = {0}, b = {1}, c = {2}", a, b, c);

            a = a + b + c;
            b = a - b - c;
            c = a - b - c;
            a = a - b - c;

            Console.WriteLine("After Swapping: ");
            Console.WriteLine("a = {0}, b = {1}, c = {2}", a, b, c);
        }
    }
}
